src/app/app/upload/_components/FileErrors.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import { useTranslations } from "next-intl";
+import { Props } from "@/types";
+
+export default function FileErrors({ errors, maxMB = 5 }: Props) {
+  const t = useTranslations("errors");
+  if (!errors?.length) return null;
+
+  return (
+    <div role="status" aria-live="polite" className="mt-4 space-y-2">
+      {errors.map((code, i) => {
+        let message: string;
+        switch (code) {
+          case "NOT_PDF":
+            message = t("NOT_PDF");
+            break;
+          case "EMPTY_FILE":
+            message = t("EMPTY_FILE");
+            break;
+          case "TOO_LARGE":
+            message = t("TOO_LARGE", { maxMB });
+            break;
+          default:
+            message = code;
+        }
```

Comment:
Check this approach, only specify the one that has special replacement, any other should be handled as default, with this approach you rely on `t()` method to validate the param and raising alerts in case of not found.
```suggestion
        switch (code) {
          case "TOO_LARGE":
            message = t("TOO_LARGE", { maxMB });
            break;
          default:
            message = t(code);
        }
```

Reply:
Right!